modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,22 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parseAndSummarise(file: str, config: Config,
+                        delete_after_parsing: bool):
+    with open(file, 'rb') as fd:
+        data = fd.read()
+        stream = DataStream(data, byte_order='big', is_signed=False)
+        try:
+            eid, summary = parsePelSummary(stream, config)
+            if(eid):
+                return eid, summary
+            else:
+                print(f"No PEL parsed for {file}")
+        except Exception as e:
+            print(f"No PEL parsed for {file}: {e}")
```

Comment:
Suggestion: Good to add "Exception" as prefix in the trace if "e" is just print exception message.